src/main/java/org/joda/beans/maven/JodaBeansValidateMojo.java

```diff
@@ -56,28 +57,33 @@ protected void runTool(
 
         logInfo("Joda-Bean validator started, directory: " + getSourceDir() +
                         (getTestSourceDir().length() == 0 ? "" : ", test directory:" + getTestSourceDir()));
-        int changes = runTool(toolClass, argsList);
-        if (changes > 0) {
+        List<File> changedFiles = runTool(toolClass, argsList);
+        if (changedFiles.size() > 0) {
             if (stopOnError) {
-                throw new MojoFailureException("Some Joda-Beans need to be re-generated (" + changes + " files)");
+                changedFiles.forEach(file -> getLog().warn("Joda-Bean needs to be re-generated: " + file));
+                throw new MojoFailureException("Some Joda-Beans need to be re-generated (" + changedFiles.size() + " files)");
             }
-            logInfo("*** Joda-Bean validator found " + changes + " beans in need of generation ***");
+            logInfo("*** Joda-Bean validator found " + changedFiles.size() + " beans in need of generation ***");
```

Comment:
Did you forget to actually log the files? AFAICT this PR has no actual effect.

Reply:
Line 63:
`changedFiles.forEach(file -> getLog().warn("Joda-Bean needs to be re-generated: " + file));`